examples/search_behavior_mbc.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+"""
+This example program shows how to search MBC
+https://github.com/MBCProject/mbc-markdown
+
+MBC as of 2023 is present in the CAPA tool integrated in VirusTotal
+
+NOTE: In order to use this script you will need to have access to
+VT Intelligence or to the Premium API. Learn more about these services at:
+https://www.virustotal.com/gui/intelligence-overview
+https://developers.virustotal.com/v3.0/reference#search
+https://www.virustotal.com/learn/
+"""
+
+
+import argparse
+import asyncio
+import datetime
+import logging
+import sys
+import vt
+
+try:
+  import yaml
+except ModuleNotFoundError:
+  print('this example uses "pyyaml" please install it. https://pyyaml.org/')
+  sys.exit(1)
+
+
+DEFAULT_PATH = 'behavior_reports'
+
+LOGGING_LEVEL = logging.INFO  # Modify if you just want to focus on errors
+logging.basicConfig(level=LOGGING_LEVEL,
+                    format='%(asctime)s %(levelname)-8s %(message)s',
+                    datefmt='%Y-%m-%d %H:%M:%S',
+                    stream=sys.stdout)
+
+
+class FetchMBCHandeler:
+  """Handler for Downloading files from VT."""
+
+  def __init__(self, apikey, num_files):
+    self.apikey = apikey
+    self.num_files = num_files
+    self.queue = asyncio.Queue()
+
+
+  async def parse_mbc_from_behavior_report(self, file_hash, behavior_report):
+    """Parse MBC rule ID from report."""
+
+    signature_matches = getattr(behavior_report, 'signature_matches', None)
+    # no signatures
+    if not signature_matches:
+      return
+
+    for sig_match in signature_matches:
+
+      if sig_match.get('format') != 'SIG_FORMAT_CAPA':
+        logging.info('unexpected rule format %s', sig_match)
+        continue
+      rule_src = sig_match.get('rule_src')
+      capa_rule = yaml.safe_load(rule_src)
+      mbc_entries = capa_rule.get('rule', {}).get('meta', {}).get('mbc', [])
+      if not mbc_entries:
```

Comment:
this `if` is not necessary, if `mbc_entries` is an empty list the `for` loop won't execute anyways.